instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/util/SpanNames.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.instrumentation.api.incubator.semconv.util;
+package io.opentelemetry.instrumentation.api.semconv.util;
```

Comment:
let's keep the old class for a few releases (maybe until 3.0), and mark it deprecated, to give people an easy path forward

Reply:
done